WolvenKit.Modkit/RED4/Tools/MaterialTools.cs

```diff
@@ -1192,6 +1192,24 @@ private static MemoryStream GetMaterialStream(Stream ms, CR2WFile cr2w)
             return new MemoryStream(b.GetBytes());
         }
 
+        private static Dictionary<string, CMaterialTemplate> GetEmbeddedMaterialTemplates(ref CR2WFile cr2w)
+        {
+            var materialTemplates = new Dictionary<string, CMaterialTemplate>();
+            foreach (var file in cr2w.EmbeddedFiles)
+            {
+                if (Path.GetExtension(file.FileName).Contains("mt"))
+                {
+                    var mt = file.Content as CMaterialTemplate;
+                    if(mt != null)
```

Comment:
Can you change the line and the one before to `if (file.Content is CMaterialTemplate mt)`?